packages/babel-plugin-lingui-macro/package.json

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "@lingui/babel-plugin-lingui-macro",
+  "version": "4.8.0-next.1",
+  "description": "Babel plugin for transforming Lingui Macros",
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "contributors": [
+    {
+      "name": "Timofei Iatsenko",
+      "email": "timiatsenko@gmail.com"
+    }
+  ],
+  "author": {
+    "name": "Tomáš Ehrlich",
+    "email": "tomas.ehrlich@gmail.com"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "license": "MIT",
+  "keywords": [
+    "babel-plugin",
+    "i18n",
+    "internationalization",
+    "i10n",
+    "localization",
+    "i9n",
+    "translation",
+    "multilingual"
+  ],
+  "scripts": {
+    "build": "rimraf ./dist && unbuild",
+    "stub": "unbuild --stub"
+  },
+  "exports": {
+    ".": {
+      "require": {
```

Comment:
this may (but prob not) require a bit of extra setup in RN. Please treat this as a note to myself, I'll document it in the RN guide if needed